src/biogeochem/CNPhenologyMod.F90

```diff
@@ -2452,8 +2470,22 @@ subroutine CNOffsetLitterfall (num_soilp, filter_soilp, &
 
             if (offset_counter(p) == dt) then
                t1 = 1.0_r8 / dt
-               leafc_to_litter(p)  = t1 * leafc(p)  + cpool_to_leafc(p)
+               !leafc_to_litter(p)  = t1 * leafc(p)  + cpool_to_leafc(p)
                frootc_to_litter(p) = t1 * frootc(p) + cpool_to_frootc(p)
+               
+               ! Y. Cheng, cut 70% AGB for switchgrass and Miscanthus and move to grainc_to_food
+               if (ivt(p)==nmiscanthus .or. ivt(p)==nirrig_miscanthus .or. ivt(p)==nswitchgrass .or. ivt(p)==nirrig_switchgrass) then
+               	   leafc_to_litter(p)  = t1 * leafc(p)*0.3  + cpool_to_leafc(p)
+               	   grainc_to_food(p) =grainc_to_food(p)+ t1 * leafc(p) * 0.7 
+               	   grainn_to_food(p) =grainn_to_food(p)+ t1 * leafn(p) * 0.7
```

Comment:
I notice that, elsewhere in this module, it looks like C fluxes are determined as they are here, but then N fluxes are determined in a more complex way. e.g.,:

```Fortran
               leafn_to_litter(p)          =  leafc_to_litter(p)/leafcn_offset(p) - leafn_to_retransn(p)
```

Do we need something like that here, or is it okay to assume that the same proportion of leafn goes to grainn as there is leafc going to grainc?

Reply:
@billsacks this is an interesting point, although I'm not sure one that @yanyanchengHydro needs to tackle. I do think that there is likely a different proportion of leafn allocated to grain than leafc, as grain typically has a higher proportion of N than in leaves (e.g., grain has lower C:N than leaves). It's probably worth looking into the literature and revisiting the grainn_to_food equation.